### PR TITLE
Add customer number for order export / import

### PR DESCRIPTION
**Pull request by migo315**
Add the customer number for order export

### CODE REVIEW

File: Components/SwagImportExport/DbAdapters/OrdersDbAdapter.php [hunk 1]
```diff
@@ -443,6 +443,7 @@ public function getDefaultColumns()
         ];
 
         $billingColumns = [
+            'billing.number as customerNumber',
```
Comment (mitelg) [inline Components/SwagImportExport/DbAdapters/OrdersDbAdapter.php]: wouldn't it make more sense to put this into the `$customerColumns` array?

Comment (migo315) [reply Components/SwagImportExport/DbAdapters/OrdersDbAdapter.php]: You are right. Just updated the commit.

File: Components/SwagImportExport/DbAdapters/OrdersDbAdapter.php [hunk 2]
```diff
@@ -490,6 +490,7 @@ public function getDefaultColumns()
             'customer.groupKey as customergroup',
             'customer.newsletter as newsletter',
             'customer.affiliate as affiliate',
+            'billing.number as customerNumber',
```
Comment (mitelg) [inline Components/SwagImportExport/DbAdapters/OrdersDbAdapter.php]: and now you could use `customer.number`, couldn't you?

Comment (migo315) [reply Components/SwagImportExport/DbAdapters/OrdersDbAdapter.php]: You are right again. I've updated the pr.

**Comment by mitelg** [comment]
thanks for your contribution @migo315  👍🎉💙

**Pull request closed without merging**